src/main/java/frc/robot/settings/Constants.java

```diff
@@ -50,8 +50,19 @@ private Climber() {
         public static final double MID_CLARM_SPEED_OUT = .7;
         public static final double TRAVERSE_CLARM_SPEED_IN = .5;
         public static final double TRAVERSE_CLARM_SPEED_OUT = .7;
-        //clarm means climber arm
-        //this has the constants for both the new and old climbers together
+        public static final double CLARM_SPEED_CALIBRATE = .3;
```

Comment:
```suggestion
        public static final double ARM_CALIBRATION_SPEED = .3;
```
Batch change for the variable name